crates/nu-path/Cargo.toml

```diff
@@ -16,6 +16,7 @@ dirs = { workspace = true }
 
 [target.'cfg(windows)'.dependencies]
 omnipath = { workspace = true }
+path_abs = "0.5.1"
```

Comment:
It's probably not a good idea to add an unsupported crate. We do however like the omnipath crate. Can it do what you're trying to use path_abs for?

Reply:
OK, I'll learn how to use omnipath, :)